src/layout/Layout/Layout.tsx

```diff
@@ -0,0 +1,16 @@
+import { Outlet } from 'react-router-dom';
+import { Header } from '../Header/Header';
+import { Footer } from '../Footer/Footer';
+import { LayoutWrapper } from './Layout.styled';
+
+export const Layout = () => {
+    return (
+        <LayoutWrapper>
```

Comment:
Is LayoutWrapper necessary? cause then the semantics will fail right.... in this case the semantics will be div->header->main->footer

Reply:
Removed.